docs/source/models/templates/concept_model_template.rst

```diff
@@ -72,188 +79,242 @@ Concept model template
+  * - Population size per draw
+    - Desired simulated population size per draw for a given simulation. 
+    - e.g. 100,000 simulants
+    - Read more about how to determine a reasonable population size for your simulation 
+      :ref:`here <vivarium_best_practices_monte_carlo_uncertainty>`. Note that engineers will
+      decide how many random seeds to use to achieve the specified population size per draw.
+  * - Age start (initialization)
+    - If sim duration > age start for observation, age_start at initialization will be zero and
+      fertility will need to be included. Otherwise, age start at initialization will be age start for 
+      observation minus simulation duration and a closed cohort (no fertility).
+    - e.g. 0 months
+    - Currently vivarium can only have new simulants enter the simulation at age zero, although 
+      changing this could be a potential framework improvement. 
+  * - Age start (observation)
+    - Age at which simulants are included in observer.
+    - e.g. 6 months
+    -
+  * - Age end
+    - Age at which simulants are no longer included in observer. 
+    - e.g. 5 years
+    -
+  * - Exit age
+    - Age at which simulants age out of the simulation. 
+    - e.g. 5 years
+    -
+  * - Simulation start date
+    - 
+    - e.g. 2021-07-01
+    -
+  * - Simulation observation start date
+    - 
+    - e.g. 2022-01-01
+    -
+  * - Simulation end date
+    - 
+    - e.g. 2026-12-31
+    -
+  * - Timestep
+    - Amount of time that passes in simulation between each "step," which is when events and observations can happen.
+    - e.g. 4 days
+    - Read more about how to determine a reasonable timestep for your simulation 
+      :ref:`here <vivarium_best_practices_time_steps>`.
+  * - Randomness key columns
+    - Parameters used to identify identical simulants across scenarios
+    - e.g. ['entrance_time', 'maternal_id', 'sex']
+    - Entrance time is currently required as a randomness key column 
   
 
+.. _{YOUR_MODEL_SHORT_NAME}2.4:
 
-**Confounders (C)**:
-
-
-
-**Effect modifiers**:
-
-
-**Mediators (M)**:
-
-
-.. _{YOUR_MODEL_SHORT_NAME}3.2:
-
-3.2 Effect sizes
-----------------
-
-
-
-4.0 Intervention
-++++++++++++++++
+2.4 Population description
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Describe the simulated population demographics here. Information in this section might include: 
+  - Cohort type
+  - Cohort length
+  - Age and sex structure
+  - Time step
+  - Fertility
+  - Stratifications 
 
+.. _{YOUR_MODEL_SHORT_NAME}2.5:
 
-.. _{YOUR_MODEL_SHORT_NAME}4.1:
+2.5 Location description
+~~~~~~~~~~~~~~~~~~~~~~~~
 
-4.1 Simulation scenarios
-------------------------
+Describe the location (country and/or region(s)) of the simulation here.
 
+.. _{YOUR_MODEL_SHORT_NAME}3.0:
 
-.. _{YOUR_MODEL_SHORT_NAME}5.0:
-
-5.0 Vivarium modelling components
-+++++++++++++++++++++++++++++++++
+3.0 Verification & validation (V&V) tracking
+--------------------------------------------
```

Comment:
Should this section be one level out, i.e., `++++++` like sections 2.0 and 4.0 instead of `------`?